crates/burn-tensor/src/tests/mod.rs

```diff
@@ -79,6 +79,8 @@ macro_rules! testgen_all {
         burn_tensor::testgen_transpose!();
         burn_tensor::testgen_tri!();
         burn_tensor::testgen_powf!();
+        burn_tensor::testgen_any!();
+        burn_tensor::testgen_all_op!();
```

Comment:
oh we had a name clash here! all_op is a good name

Reply:
For consistency, I would have renamed `testgen_all` to something else. Maybe `testgen_every_op`?

Reply:
Not a bad idea, but goes beyond the scope of this PR. all_op is good for now